content/actions/managing-issues-and-pull-requests/scheduling-issue-creation.md

```diff
@@ -19,9 +19,9 @@ topics:
 
 ## Introduction
 
-This tutorial demonstrates how to use the [`imjohnbo/issue-bot` action](https://github.com/marketplace/actions/issue-bot-action) to create an issue on a regular basis. For example, you can create an issue each week to use as the agenda for a team meeting.
+This tutorial demonstrates how to use the [GitHub CLI](https://cli.github.com/) to create an issue on a regular basis. For example, you can create an issue each week to use as the agenda for a team meeting.
```

Comment:
nit to avoid in-line links for the style guide and to use a variable name

```suggestion
This tutorial demonstrates how to use the {% data variables.product.prodname_cli %} to create an issue on a regular basis. For example, you can create an issue each week to use as the agenda for a team meeting. For more information about {% data variables.product.prodname_cli %}, see "[AUTOTITLE](/actions/using-workflows/using-github-cli-in-workflows)."
```